daemon/policy.go

```diff
@@ -93,11 +93,53 @@ func (d *Daemon) TriggerPolicyUpdates(added []policy.NumericIdentity) {
 			if policyChanges {
 				ep.Regenerate(d)
 			}
+			// Only update whether policy is enabled after rules are added successfully for this endpoint.
+			d.updatePolicyEnforcement(ep)
```

Comment:
I figured we would update the flags only if policy enforcement was successful. What would the motivation be for updating at the beginning of the loop?